src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,80 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+import { MovieData } from '../../types/MovieData';
+
+type Props = {
+  onMovies: (m: Movie | null) => void,
+};
+
+export const FindMovie: React.FC<Props> = ({
+  onMovies,
+}) => {
+  const [title, setTitle] = useState('');
+  const [selectedMovie, setSelectedMovie] = useState<Movie | null>(null);
+  const [hasFindError, setHasFindError] = useState(false);
+  const [hasLoadingSubmit, setHasLoadingSubmit] = useState(false);
+  const [hasFirstLoad, setHasFirstLoad] = useState(true);
+
+  const createNewMovie = (movieDate: MovieData) => {
+    const newDescrip = movieDate.Plot === 'N/A'
+      ? 'Unfortunately there is no description'
+      : movieDate.Plot;
+    const newposter = movieDate.Poster === 'N/A'
```

Comment:
Avoid using magic strings like 'N/A'. Consider defining it as a constant at the top of your file or in a separate constants file.